### ISSUE TITLE
Little Nightmares (424840)

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Proton#1324]
# Compatibility Report
- Name of the game with compatibility issues: Little Nightmares
- Steam AppID of the game: 424840

## System Information
- GPU: Iris Pro 580
- Driver/LLVM version: Mesa 18.0.5
- Kernel version: 4.15
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/daasuharo/b18e39c1f39b372a3e73892748c2da28
- Proton version: 3.7-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

 
[steam-424840.log](https://github.com/ValveSoftware/Proton/files/2363382/steam-424840.log)

## Symptoms  
![screenshot from 2018-09-08 08-35-47](https://user-images.githubusercontent.com/1419625/45255338-38629b80-b342-11e8-95df-f4c28d361462.png)
Shows that when trying to load the game.

## Reproduction
Try to load the game.

**Comment by ah-** [comment]
This starts for me when ran with the "-onethread" argument.

**Comment by lucifertdark** [comment]
If anyone else is getting crackling noises while playing, installing xact with winetricks will sort those out.

**Comment by ah-** [comment]
Yep, can confirm @lucifertdark. With -onethread and "protontricks 424840 xact" it runs perfectly for me.

From debugging it, seems like there's a threading issue that results in a null pointer in the render thread during startup. After looking around the internet it also seems to affect Windows, so it might even be a genuine bug in the game that just happens to affect wine more reliably. Not sure if we're losing any performance due to running it with -onethread, still at 60 fps for me.

**Comment by NoXPhasma** [comment]
Using `-onethread` works for me as well, however I do have crackling audio, even with `xact` installed.

**Comment by lucifertdark** [comment]
Setting xaudio2_6 & xaudio2_7 to Native may help to fix the crackling.

**Comment by NoXPhasma** [comment]
This works, thank you!

**Comment by NTMan** [comment]
On [my setup](https://gist.github.com/NTMan/33cca4906328793206218192820779ec) game immediately crashes after start.
[steam-424840.log](https://github.com/ValveSoftware/Proton/files/2833570/steam-424840.log)

![screenshot from 2019-02-06 00-39-47](https://user-images.githubusercontent.com/200750/52299763-18460f80-29a8-11e9-9bfd-af3dae309571.png)

**Comment by simifor** [comment]
Fresh proton 8.0-3 prefix, the game launched fine without extra launch arguments, music worked fine too. There was however an issue with the aspect ratio (my monitor is 16:10)
![Little Nightmares  2023-09-02_17:52:17](https://github.com/ValveSoftware/Proton/assets/70454800/b9cc1529-4c7f-444a-9cb4-c318c7c58418)
Turning off fullscreen fixed this issue, and after doing that and turning it on back again the issue didn't resurface
![Little Nightmares  2023-09-02_17:53:38](https://github.com/ValveSoftware/Proton/assets/70454800/65cd989c-23e5-44a6-b3b1-fc7d1b30f02b)
Weirdly enough, I wasn't able to reproduce the aspect ratio issue, I tried deleting the configuration folder, and later deleting the prefix, but the game started normally both times.

**Comment by opyate** [comment]
Not an issue, just something that happened:

After biting the ghost's neck, half way through the next cutscene I got a popup saying "fatal error", then I rebooted the deck and now when I start the game, I see this.

![Image](https://github.com/user-attachments/assets/3eadce3d-753b-4a8f-aca9-4503a65e0341)

Trying again, I managed to resume my game, but had to do the ghost/mirror level all over again.

Managed to complete the game after that.
